girder_annotation/girder_large_image_annotation/models/annotation.py

```diff
@@ -508,7 +508,7 @@ class AnnotationSchema:
             'group': groupSchema,
         },
         'required': ['girderId', 'type'],
-        'additionalProperties': False,
+        'additionalProperties': True,
```

Comment:
Probably the correct thing to do is to make the overlay annotation inherit from the baseShapeSchema, except that we don't want to get the line and lineWidth properties.  We can do this in a separate PR -- baseShapeSchema would inherit from baseElementSchema which would exclude those properties and then the overlay annotation would inherit from that.

Reply:
Created #762